src/scripts.js

```diff
@@ -28,13 +30,24 @@ const ingredientSidebar = document.querySelector("#ingredientSection")
 const userName = document.querySelector('#user-info');
 const favoritesView = document.querySelector('#favorites-view');
 const savedButton = document.querySelector('#saved-recipe-button');
+const saveRecipeButton = document.querySelector('#favorite-recipe-button')
+const homeButton = document.querySelector('#home-button')
+const submitButton = document.querySelector('#submit-search-button')
```

Comment:
we can change submitButton name to submitTextButton since we'll have another submit button for the radio buttons

Reply:
@knpausch , good catch! We will keep in mind today as we merge our two PRs to main!!